src/pages/index.js

```diff
@@ -104,3 +104,31 @@ addCardButton.addEventListener("click", () => {
   addFormValidator.toggleButtonState();
   newFormPopup.open();
 });
+
+fetch("https://around-api.en.tripleten-services.com/v1", {
+  headers: {
+    authorization: "56e30dc-2883-4270-a59e-b2f7bae969c6",
+  },
+})
+  .then((res) => res.json())
+  .then((result) => {
+    console.log(result);
+  });
+
```

Comment:
Needs correction: You should implement all the api requests according to the task (update user, add card, remove card, like card, dislike card, update avatar)